src/streaming/controllers/ProtectionController.js

```diff
@@ -439,20 +496,26 @@ MediaPlayer.dependencies.ProtectionController = function () {
             this[MediaPlayer.models.ProtectionModel.eventList.ENAME_KEY_STATUSES_CHANGED] = onKeyStatusesChanged.bind(this);
 
             keySystems = this.protectionExt.getKeySystems();
-            this.protectionModel = this.system.getObject("protectionModel");
-            this.protectionModel.init();
+
+            // For browsers that don't support any version of EME, the app crashes without this check
+            if (this.system.hasMapping("protectionModel")) {
+                this.protectionModel = this.system.getObject("protectionModel");
+                this.protectionModel.init();
+            }
 
```

Comment:
@rashmimittal ProtectionModel is only instantiated by ProtectionController.  ProtectionControllers can only be created (by an application) by calling `MediaPlayer.createProtection()`.  Applications do not have access to the Dijon system object for instantiating new instances, which is why we had to add the MediaPlayer API.  A preventative call in `MediaPlayer.createProtection()` to check for EME support would suffice.  However, this goes back to the ES6 refactor, in which we may no longer be using Dijon.